.changeset/mighty-ways-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+Add Open Collective social icon
```

Comment:
Minor typo: update the changeset more to present tense? Reference: [Astro Docs Docs](https://contribute.docs.astro.build/guides/new-feature-docs/#changesets)
```suggestion
Adds Open Collective social link icon
```
Icon is working 💯 on [Preview](https://starlight-d5nl6x0n4-astrodotbuild.vercel.app/guides/components/#all-icons)